Samples/auth/Outlook-Add-in-SSO/README.md

```diff
@@ -146,61 +94,51 @@ The browser will attempt to redirect back to your app, which may not be running.
 
 ## Run the Sample
 
-1. Select the **AttachmentDemo** project in **Solution Explorer**, then choose the **Start Action** value you want (under **Add-in** in the properties window). Choose any installed browser to launch Outlook on the web, or you can choose **Office Desktop Client** to launch Outlook on Windows. If you choose **Office Desktop Client**, be sure to configure Outlook to connect to the Office or Outlook.com user you want to install the add-in for.
+1. In the **Solution Explorer**, right-click the **AttachmentDemo** project and select **Properties**.
+1. In the **Properties** window, set the following property values.
 
-1. Press **F5** to build and debug the project. You may be prompted to trust the developer certificate.
+| Property | Value |
+|----------|-------|
+| `Start Action` | `Office Desktop client` or an installed browser. |
+| `Email Address` | The email address of a user in your test tenant. |
+| `Use multi-factor auth` | `True` |
 
+1. Press **F5** to build and debug the project. You may be prompted to trust the developer certificate.
 1. You should be prompted for a user account and password. Provide a user in your Office tenant, or an Outlook.com account. The add-in will be installed for that user, and either Outlook on the web or Outlook on Windows will open.
 
 1. Select any message, **that has one or more attachments**.
-
 1. Open the task pane:
-    
-    * If you're in Outlook on the web: Select the **...** (**More actions**) drop down menu, and then choose **AttachmentDemoWeb**.<br>
-![Screen shot of the elipses button in Outlook on the web](buttons-outlook-web.png)
-    * If you're in Outlook on Windows or Mac: On the **Home** tab, select **Choose attachments**. Note that if the Outlook app window is too small, that **Choose attachments** will instead be located on the **Home** tab's **...** (**More commands**) button.<br>
-![Screen shot of the Choose attachments button on Home tab in Outlook on the web](buttons-outlook-desktop.png)
-    
-1. In the **AttachmentDemoWeb** task pane that opens, select the attachments you want to save.
 
-1. Choose **Save selected**
+    - If you're in Outlook on the web: Select the **...** (**More actions**) drop down menu, and then choose **AttachmentDemoWeb**.
+
+    ![Screen shot of the elipses button in Outlook on the web](buttons-outlook-web.png)
```

Comment:
This screenshot isn't loading in the PR.
![image](https://user-images.githubusercontent.com/38896772/207173227-6de6063b-be73-402d-a4d5-0bd17041b9ec.png)

Reply:
Should be fixed.